### DISCUSSION TITLE
Timezone CLI command

### DISCUSSION THREAD

**Discussion by Devinator** [WLAN-Pi/feedback#18] [Feature Request]
Awesome Developers,

https://userguide.wlanpi.com/hardware/wlan-pi-pro/getting-started/web-front-end/cockpit-admin-gui#timezone-update 

This page doesn't show the timezone command, and the FPMS Date/Time option doesn't allow for any configuration.

Thanks!

Devin

**Comment by jolla** [comment]
Ability to change timezone from FPMS would be nice, I've added your request to the internal feature request list, we'll try to get it added to the next release. Thanks!
